pkg/integration/components/input.go

```diff
@@ -76,6 +76,12 @@ func (self *Input) Confirm() {
 	self.pressKey(self.keys.Universal.Confirm)
 }
 
+func (self *Input) ProceedWhenAsked(matcher *matcher) {
```

Comment:
adding this because we do this all over the place

Reply:
Smart

Reply:
https://www.youtube.com/watch?v=Ll9nfNlDLII&ab_channel=RapPadB